packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -178,7 +179,7 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
     }
   }
 
-  handleScrollButtons = () => {
+  handleScrollButtons = _.debounce(() => {
```

Comment:
Echoing what I've been told recently here, which is lodash isn't something we want as a dependency going forward; https://github.com/patternfly/patternfly-react/issues/6553.

So I would suggest using `setTimeout` instead to accomplish the debouncing of this operation.

Reply:
@jeffpuzzo Thanks, I will remove the use of lodash.